inference-engine/tests/functional/plugin/myriad/shared_tests_instances/single_layer_tests/eltwise.cpp

```diff
@@ -39,7 +43,9 @@ std::vector<ngraph::helpers::EltwiseTypes> eltwiseOpTypes = {
         ngraph::helpers::EltwiseTypes::ADD
 };
 
-std::map<std::string, std::string> additional_config = {};
+std::map<std::string, std::string> additional_config = {
+    {VPU_CONFIG_KEY(DETECT_NETWORK_BATCH), CONFIG_VALUE(NO)}
```

Comment:
Yes, this was added to make the added tests work. Without it, the GT will try to reshape the networks as the first input has batch>1, but it will fail on reshaping of second input, because it has only one dimension. We have a ticket on how to make GT smarter on his decision whether to reshape the network or not, but now the only one way for tests is to set config value to disable this logic at all.

Reply:
Ok. It would be nice for me to add the ticket number near to `additional_config`.

Reply:
Well, actually I don't think that it is necessary because this config option is widely used among all the tests, so it would be not enough to add ticket's number just here, but to all of the places. If it will be going to be fixed someday it will be better to just look through all of the places where this config is used (you can see yourself, mostly all of its uses are in tests and for the same purpose).